doc/user_guide/configuration/all-options.rst

```diff
@@ -202,6 +195,13 @@ Standard Checkers
 **Default:**  ``True``
 
 
+--source-roots
```

Comment:
Is this meant to be part of this MR?

Reply:
We forgot to generate the doc previousely because of https://github.com/PyCQA/pylint/pull/6736